code/modules/research/techweb/all_nodes.dm

```diff
@@ -111,7 +111,7 @@
 	description = "Finely-tooled manufacturing techniques allowing for picometer-perfect precision levels."
 	prereq_ids = list("engineering", "datatheory")
 	design_ids = list("pico_mani", "super_matter_bin")
-	research_cost = 2500
+	research_cost = 7500

```

Comment:
Lowered from  10K since this is literally the only 2 in here, but not 5K due to their importance